src/components/NavigationDrawer.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { StyleSheet, View, Image, TouchableOpacity } from 'react-native'
```

Comment:
@Harjot1Singh i've been using native-base for RN components. Are we happy to use that lib or do you have any preference on any others?

Reply:
This was done for placeholders. When tackling #59 will make sure that `native-base` is used for most parts.